src/vm/dllimportcallback.cpp

```diff
@@ -1167,6 +1167,36 @@ VOID UMEntryThunk::FreeUMEntryThunk(UMEntryThunk* p)
 
 #endif // CROSSGEN_COMPILE
 
+VOID __fastcall UMEntryThunk::ReportViolation(UMEntryThunk* pEntryThunk)
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_TRIGGERS;
+        MODE_COOPERATIVE;
+        PRECONDITION(CheckPointer(pEntryThunk));
+    }
+    CONTRACTL_END;
```

Comment:
I would be useful to add a comment here that this diagnostic is best effort, it won't report the problem in 100% of cases, and it may sometime crash while trying to report the problem.